lua/starfall/libs_sh/entities.lua

```diff
@@ -930,6 +932,16 @@ if SERVER then
 		checkluatype(val, TYPE_NUMBER)
 		ent:SetMaxHealth(val)
 	end
+		
+	--- If true, stops the entity from being saved on duplication or map save.
+	-- @server
+	-- @param boolean dontduplicate True to disallow duplication.
+	function ents_methods:doNotDuplicate(duplicate)
```

Comment:
It shouldn't take a param because then users can make scripted entities that shouldn't be duped dupable.